static/menu.html

```diff
@@ -30,8 +30,8 @@
         <div class="dialog-content" id="rules-content">
             <h3 id="rules-heading">נחשו את המילה הסודית</h3>
             <p>
-            אפשר לנחש מילה או ביטוי קצר.
-            המילה הסודית יכולה להיות בכל אורך, אבל תהיה מילה בודדת (ולא ביטוי).
+            אפשר לנחש כל מילה בעברית.
```

Comment:
Thanks!

How about adding an elaboration regarding what "בעברית" means? The secret word in Hebrew-Semantle #4 was "דנטלי" and many people didn't consider it as a Hebrew word (and there might be such words in the future)

Reply:
That's an interesting point... I'll try to mention it while staying short and to the point :)

Reply:
I brought this up in conversion. My dad suggested to change it to "any word in the Hebrew lexicon" (כל מילה בלקסיקון העברי or באוצר המילים בעברי). Either way, my parents didn't like my suggested phrasing (בעברית או לועזית), so perhaps you shouldn't merge yet!